crates/diesel_models/src/query/merchant_account.rs

```diff
@@ -121,6 +121,26 @@ impl MerchantAccount {
         .await
     }
 
+    pub async fn list_all_merchant_accounts(
+        conn: &PgPooledConn,
+        limit: Option<u32>,
+        offset: Option<u32>,
+    ) -> StorageResult<Vec<Self>> {
```

Comment:
Instead of `Self`, can we get only `merchant_id` and `org_id`, which we don't have to decrypt?

Reply:
For the interface function we are getting merchant and org ids, the query is generic to fetch merchant accounts and it has limit as well. Moreover we are not decrypting merchant accounts to get org_id and merchant ids.